src/modules/default/template/mup.js.sample

```diff
@@ -21,6 +21,8 @@ module.exports = {
     env: {
       ROOT_URL: 'app.com',
       MONGO_URL: 'mongodb://localhost/meteor',
+      PORT: 80,
+      BIND: '0.0.0.0',
     },
```

Comment:
Most apps won't need to change these, so I would prefer to not have them in the default mup.js.